src/components/Main/Main.module.scss

```diff
@@ -0,0 +1,4 @@
+.wrap {
+  display: flex;
+  min-height: calc(100vh - 100px);
```

Comment:
I think it will be problem when content will be bigger than height of the page.
Show me screenshot with scrollbar